django/db/models/query.py

```diff
@@ -1190,11 +1190,18 @@ def update(self, **kwargs):
         # Inline annotations in order_by(), if possible.
         new_order_by = []
         for col in query.order_by:
-            if annotation := query.annotations.get(col):
+            new_col = col
+            descending = False
+            if isinstance(col, str) and col.startswith('-'):
+                new_col = col[1:]
+                descending = True
+            if annotation := query.annotations.get(new_col):
```

Comment:
```suggestion
            alias = col
            descending = False
            if isinstance(alias, str) and alias.startswith("-"):
                new_col = alias.removeprefix("-")
                descending = True
            if annotation := query.annotations.get(alias):
```

Reply:
I noticed that `new_col = alias.removeprefix("-")` This code seems useless, should it be changed to `alias = alias.removeprefix("-")`, or is there another naming convention?

Reply:
Yes, sorry typo: `alias = alias.removeprefix("-")`.